dbhydra/src/abstract_table.py

```diff
@@ -239,7 +239,12 @@ def __init__(self, db1, name, columns = None, types: Optional[list[str]] = None,
         self.types: list[str] = types #[] if types is None else types - is wrong, if types not initialized it should be None, not empty list
         self.id_column_name: str = id_column_name
         if self.columns is not None and self.types is not None:
-            assert len(self.columns) == len(self.types) 
+            assert len(self.columns) == len(self.types)
+            
+            # Hotfix: flatten nested lists (otherwise crashes might happen)
+            # TODO: Search for the causes and implement better handling
+            for i, column in enumerate(self.columns):
+                self.columns[i] = column[0] if type(column) == list else column
```

Comment:
I dont like this, why should the information be truncated? When can this even happen that column from columns would contain a list inside?